lk2nd/device/menu/menu.c

```diff
@@ -203,8 +203,15 @@ void display_fastboot_menu(void)
 		fbcon_puts_ln(SILVER, y, incr, true, "Short press to navigate.");
 		fbcon_puts_ln(SILVER, y, incr, true, "Long press to select.");
 	} else {
-		fbcon_puts_ln(SILVER, y, incr, true, "Volume keys to navigate.");
-		fbcon_puts_ln(SILVER, y, incr, true, "Power key to select.");
+		if(lk2nd_dev.menu_keys.navigate)
+			fbcon_printf_ln(SILVER, y, incr, true, "%s to navigate.", lk2nd_dev.menu_keys.navigate);
+		else
+			fbcon_puts_ln(SILVER, y, incr, true, "Volume keys to navigate.");
```

Comment:
nit but I think writing it like this is a little bit easier to read:

```suggestion
		fbcon_printf_ln(SILVER, y, incr, true, "%s to navigate.",
			    (lk2nd_dev.menu_keys.navigate ? lk2nd_dev.menu_keys.navigate : "Volume keys"));
```

Reply:
(to note, I implied fixing both this and below one, and maybe align the second line more nicely as my GH message editor doesn't have monospace enabled)

Reply:
Yes, I figured :) I already have it fixed and formatted on my local branch. Thanks!